src/components/popups-etc/tutorial-warning.tsx

```diff
@@ -0,0 +1,39 @@
+import { Signal } from '@preact/signals'
+import Button from '../design-system/button'
+import styles from './tutorial-warning.module.css'
+
+export interface TutorialWarningModalProps {
+	showingTutorialWarning: Signal<boolean>
+	exitTutorial: () => void
+}
+
+export default function TutorialWarningModal(props: TutorialWarningModalProps) {
+	return (
+		<div class={styles.overlay}>
```

Comment:
Would like to see clicking the overlay close the popup as well.

Reply:
I copied the existing draft-modal which does not have the feature. Should I add it?

Reply:
I think so! Draft modal doesn't on purpose, but I think it would make sense on this modal.